Projects.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Here we import the Boostrap CSS stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/main.css"> 
+    <link rel="stylesheet" href="./css/Projects.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
+    <title>Responsive grid project</title>
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top p-4">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="./index.html"><b>Maya Nelson</b> / PROJECT MANAGER</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse " id="navbar">
+          <ul class="navbar-nav ms-auto">
+            <li class="nav-item">
+              <a class="nav-link active"  href="./index.html">ABOUT ME</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active"  href="./Resume.html">RESUME</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="./Projects.html">PROJECTS</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active" href="./Contact.html">CONTACT</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div id="navholder"></div>
+    <main>
+      <div>
+        <h1 class="d-flex justify-content-center mt-4 py-3 ">projects</h1>
+        <div class="row mb-5">
```

Comment:
To avoid the space between the text and the images here, you can apply Bootstrap's `g-0` class to this row. This stands for "gutter zero" which removes the spaces between the columns.
(this was previously known as `no-gutters` in Bootstrap 4 but has been changed to `g-0` in Bootstrap 5)